risc0/zkvm/src/host/client/env.rs

```diff
@@ -302,28 +296,6 @@ impl<'a> ExecutorEnvBuilder<'a> {
         self
     }
 
-    /// Add a handler for simple I/O handling.
-    pub fn slice_io(&mut self, channel: &str, handler: impl SliceIo + 'a) -> &mut Self {
-        self.inner
-            .slice_io
-            .borrow_mut()
-            .with_handler(channel, handler);
-        self
-    }
-
-    /// Add a handler for simple I/O handling.
-    pub fn io_callback<C: AsRef<str>>(
```

Comment:
It might be nice to still provide this interface on-top of the PosixIo (if that even still makes sense)